libs/token.js

```diff
@@ -22,17 +19,13 @@ export function decodeToken(httpEvent) {
 /**
  * Asynchronously sign a given payload into a JSON Web Token.
  * @param {obj} jsonToSign the payload of the json web token.
- * @param {string} secretKey
+ * @param {string} secret the secret key to sign the JSON Web Token.
+ * @param {number} expireTimeInSeconds the expire time for the token in seconds.
  */
-export async function signToken(jsonToSign) {
-  const [error, secret] = await to(
-    secrets.get(config.token.secret.name, config.token.secret.keyName)
-  );
-  if (error) throwError(error.code, error.message);
-
-  // Add expiration time to JWT token, 15 min.
+export async function signToken(jsonToSign, secret, expireTimeInSeconds) {
+  // Add expiration time to JWT token.
   // The format is in seconds since Jan 1, 1970, not milliseconds, to match the default iat format of JWT.
-  jsonToSign.exp = parseInt(Date.now() / 1000) + 15 * 60;
+  jsonToSign.exp = parseInt(Date.now() / 1000) + expireTimeInSeconds * 60;
```

Comment:
You say expireTimeInSeconds, but then you multiply it with 60, so it should probably be expireTimeInMinutes, no?